### PR TITLE
[moonbase] Moonbase Strategy

### PR DESCRIPTION
**Pull request by silverstar679**

### CODE REVIEW

**Review by ChaituVR — CHANGES_REQUESTED** [review]
@silverstar679 Tests are failing can take a look?

**Comment by silverstar679** [comment]
Fixed the error. 
Please check it out and hope to merge soon.
Thanks for your support.

File: src/strategies/moonbase/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      "name": "moonbase",
+      "params": {
+        "address": "0xc97c478Fc35d75b51549C39974053a679A5C67E1",
+        "masterChef": "0x830304d6C669d33738c7E4c1F2310CC1E530Df63",
+        "moonbaseLPs": [
+          {
+            "address": "0x230C64C42886A1F6b91eD8C11B59a2D45865d38F",
+            "pid": 7
+          }
+        ],
```
Comment (ChaituVR) [inline src/strategies/moonbase/README.md:22]: Could you add a limit to `moonbaseLPs` to avoid memory issues, return an error of add a JSON schema

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/moonbase/README.md (hunk 1 above)
Comment (silverstar679) [reply src/strategies/moonbase/README.md:22]: How can I fix this error?

Comment (silverstar679) [reply src/strategies/moonbase/README.md:22]: actually all tests passed. So what should I do for this issue?

**Pull request closed without merging**